plugins/MumbleAPI_latest.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021-2022 The Mumble Developers. All rights reserved.
```

Comment:
I am wondering whether this header might be better created in `scr/mumble` instead. After all, this header is not really meant to be included by plugin authors, so I would completely move it out of their usual scope and simply into Mumble's main source code base.

In that case, we could also rename the file to e.g. `MumbleAPI_structs.h`.